core/src/main/scala/com/salesforce/op/evaluators/OpBinaryClassificationEvaluator.scala

```diff
@@ -112,10 +114,29 @@ private[op] class OpBinaryClassificationEvaluator
       val falsePositiveRateByThreshold = sparkMLMetrics.roc().collect().map(_._1).slice(1, thresholds.length + 1)
       val aUROC = sparkMLMetrics.areaUnderROC()
       val aUPR = sparkMLMetrics.areaUnderPR()
+
+      // Solutions can be checked with any system of equations solver. For Wolfram Alpha, input:
+      // inv {{(1-P), -P, 0, 0}, {(1-R), 0, 0, -R}, {0, 1-"Fpr", -"Fpr", 0}, {1,1,1,1}} * {0, 0, 0, "Count"}
+      val (tupPos, tupNeg) = thresholds.zipWithIndex.map{ case (_, i) => {
+        val denom = 1.0 / recallByThreshold(i) + (1.0 / precisionByThreshold(i) - 1.0) / falsePositiveRateByThreshold(i)
+        val prefactor = count / denom
+        val tpByThreshold = math.round(prefactor).toDouble
+        val fpByThreshold = math.round(prefactor * (1.0 / precisionByThreshold(i) - 1.0)).toDouble
+        val tnByThreshold = math.round(prefactor * (1.0 / falsePositiveRateByThreshold(i) - 1.0) *
+          (1.0 / precisionByThreshold(i) - 1.0))toDouble
```

Comment:
missing `.` for `.toDouble`, although that's not a syntax error apparently. 😮

Reply:
Oh wow, how did I miss that?

Reply:
Removed!